<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="pera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Geometr415 Blk BT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B4CDCD;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Escolher o plano:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cadastrar">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>310</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GenÃªro:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>56</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Idade:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linha1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linha2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>47</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Peso:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>59</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="linha4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>120</y>
        <width>141</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="sexfemin">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Feminino</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sexmasc">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Masculino</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Altura:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linha3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
     </widget>
     <widget class="QLabel" name="teste">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>230</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="caixa">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Pessoas Cadastradas</string>
     </attribute>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>761</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
       <property name="font">
        <font>
         <family>Geometr415 Blk BT</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Idade</string>
       </property>
       <property name="font">
        <font>
         <family>Geometr415 Blk BT</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Altura</string>
       </property>
       <property name="font">
        <font>
         <family>Geometr415 Blk BT</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peso</string>
       </property>
       <property name="font">
        <font>
         <family>Geometr415 Blk BT</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>GÃªnero</string>
       </property>
       <property name="font">
        <font>
         <family>Geometr415 Blk BT</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Imc</string>
       </property>
       <property name="font">
        <font>
         <family>Geometr415 Blk BT</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>367</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="porNome">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#D1EEEE;</string>
         </property>
         <property name="text">
          <string>Ordenar por nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="porImc">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#D1EEEE;</string>
         </property>
         <property name="text">
          <string>Ordenar por Imc</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Page">
     <attribute name="title">
      <string>Detalhes</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pesquisar UsuÃ¡rio:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pesquisar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Plano</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Imc</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="recomendacao">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>491</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Geometr415 Blk BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D1EEEE;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>EstatÃ­sticas</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>671</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pessoas cadastradas:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mostrar1">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#D1EEEE;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pessoas acima do peso:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mostrar2">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#D1EEEE;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Maior Imc:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mostrar3">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#D1EEEE;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total de dinheiro:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mostrar4">
         <property name="font">
          <font>
           <family>Geometr415 Blk BT</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#D1EEEE;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
